main.dart

```diff
@@ -0,0 +1,163 @@
+import 'package:flutter/material.dart';

```

Comment:
Why there is no flutter project structure followed? I just see dart files and py files all in same folder. Use a proper project template.